rust/clients/cli/README.md

```diff
@@ -26,6 +26,13 @@ The following commands are available:
 * user accounts
```

Comment:
There needs to be an option to show all available user accounts on this machine.
The once without password must only be displayed in local settings.